### DISCUSSION TITLE
A district is being returned as a city in the UK

### DISCUSSION THREAD

**Discussion by dyxyl** [osm-search/Nominatim#2490] [Geocoding around the world]
## What did you search for?

https://nominatim.openstreetmap.org/reverse.php?lat=52.036130&lon=0.732651&zoom=18&format=jsonv2

## What result did you get?
`
{"place_id":135388732,"licence":"Data © OpenStreetMap contributors, ODbL 1.0. https://osm.org/copyright","osm_type":"way","osm_id":147151545,"lat":"52.036130045075154","lon":"0.7326509141789389","place_rank":26,"category":"highway","type":"unclassified","importance":0.09999999999999998,"addresstype":"road","name":null,"display_name":"Sudbury, Babergh, Suffolk, East of England, England, CO10 2FA, United Kingdom","address":{"suburb":"Sudbury","town":"Sudbury","city":"Babergh","county":"Suffolk","state_district":"East of England","state":"England","postcode":"CO10 2FA","country":"United Kingdom","country_code":"gb"},"boundingbox":["52.0356408","52.0367455","0.7324105","0.7328465"]}
`
## What result did you expect?

I did not expect `city":"Babergh"`. "Babergh" is a district (https://www.openstreetmap.org/relation/53322).

## Further details

Other nearby searches return similar spurious "city" results.  In the UK, a town very unlikely to be contained within a city.

**Comment by lonvia** [comment]
That's for the most part a data issue. Tagging of administrative boundaries in the UK is a bit of mess and there is only so much Nominatim can do to take guesses at what might be meant.

I'll move this in the discussion section and leave it to UK OSMers to further discuss what the best rules for assigning administrative boundaries to address parts are.

**Comment by dyxyl** [comment]
OK, but guessing that something is a "city" as well as identifying a town is wrong in every case I have tried. Hopefully the 69 places listed in https://en.wikipedia.org/wiki/List_of_cities_in_the_United_Kingdom are correctly tagged as being cities already. I would change it to never guess a city.
